packages/core/src/Badge/Badge.stories.js

```diff
@@ -1,26 +1,26 @@
 import React from 'react'
-import { Cartesian } from '@compositor/kit'
 
 import { Badge } from '..'
 
-const sizes = { small: 'small', medium: 'medium' }
-const colors = {
-  primary: 'primary',
-  secondary: 'secondary',
-  text: 'text',
-  success: 'success',
-  error: 'error',
-  warning: 'warning',
-  alert: 'alert',
-  caution: 'caution',
-  notify: 'notify',
-  pricePrimary: 'pricePrimary',
-  priceSecondary: 'priceSecondary',
-  promoPrimary: 'promoPrimary',
-  promoSecondary: 'promoSecondary',
-  border: 'border',
-  background: 'background',
-}
+// for Args
+// const sizes = { small: 'small', medium: 'medium' }
+// const colors = {
+//   primary: 'primary',
+//   secondary: 'secondary',
+//   text: 'text',
+//   success: 'success',
+//   error: 'error',
+//   warning: 'warning',
+//   alert: 'alert',
+//   caution: 'caution',
+//   notify: 'notify',
+//   pricePrimary: 'pricePrimary',
+//   priceSecondary: 'priceSecondary',
+//   promoPrimary: 'promoPrimary',
+//   promoSecondary: 'promoSecondary',
+//   border: 'border',
+//   background: 'background',
+// }
```

Comment:
Still need this?

Reply:
keeping it around as a reminder to convert it to Args